lib/install/phases/postsystem.go

```diff
@@ -273,74 +270,3 @@ func (p *rbacExecutor) PreCheck(ctx context.Context) error {
 func (*rbacExecutor) PostCheck(ctx context.Context) error {
 	return nil
 }
-
```

Comment:
Moved the implementation to a separate file that handles both Kubernetes and Gravity resource setup.